docs/models.yml

```diff
@@ -1163,7 +1173,18 @@ speaker:
   weight:
     type: number
     default: 10000
-  marked: boolean
+  marked:
+    type: boolean
+    default: False
+  pro_speech:
+    type: boolean
+    default: False
+  contra_speech:
+    type: boolean
+    default: False
```

Comment:
I don't like the split into two fields, it just makes room for undefined datastore content and situations. Why not something like
```
speech_status:
  type: string
  enum:
    - pro
    - contra
```
and if it's null, no status is set?

Reply:
In combination with the second point in https://github.com/OpenSlides/OpenSlides/issues/6035 I would propose to do this:
```
speech_status:
  type: string
  enum:
    - none
    - pro
    - contra
    - contribution
```

I think we need the `none` case, since null is also not allowed here if I'm correct

Reply:
If the field is not required, `None` should be allowed. The other problem with your suggestion is that `none` would be interpreted as a string which means we would save the literal string `"none"` in the datastore and I hope you are not seriously suggesting this :D

Besides that point, that seems fine. I did not look closely into #6035 but if that warrants its own enum value, that is exactly the way to do it.

Reply:
> If the field is not required, None should be allowed

ok it is fine than